src/main/resources/swagger.api/dcb_transaction.yaml

```diff
@@ -6,148 +6,124 @@ info:
 paths:
   /transactions/{dcbTransactionId}:
     description: ReShare Circulation Transaction endpoint
-    parameters:
-      - in: path
-        name: dcbTransactionId
-        description: The ReShare DCB ID for the transaction being brokered.
-        required: true
-        schema:
-          type: string
     post:
       description: Create circulation request
       operationId: createCirculationRequest
       tags:
         - circulation
+      parameters:
+        - $ref: '#/components/parameters/dcbTransactionId'
```

Comment:
Hi @Vignesh-kalyanasundaram ,
As per the ongoing discussion there is going to be a library role as part of the POST request. Please keep that provision over here in the POST request. 
CC: @steveellis , @kalibek-epam